definitions/install.yaml

```diff
@@ -2666,6 +2666,16 @@ spec:
         app.kubernetes.io/part-of: kyverno
         app.kubernetes.io/version: v1.4.0-beta1
     spec:
+      affinity:
```

Comment:
Let's change the base file `definitions/manifest/deployment.yaml` and run Kustomize via the Make command to propagate the change to all files.